www/include/options/accessLists/reloadACL/reloadACL.php

```diff
@@ -42,18 +42,29 @@
 require_once "./include/common/common-Func.php";
 require_once "./class/centreonMsg.class.php";
 
-if (isset($_GET["o"]) && $_GET["o"] == "r") {
+$action = filter_var(
+    $_GET['o'] ?? $_POST['o'] ?? null,
+    FILTER_SANITIZE_STRING
+);
+
+/*
+ * $action == r : when using the reload button
+ * $action == u :
+ */
+if ($action === "r") {
     $sid = session_id();
-    $pearDB->query("UPDATE session SET update_acl = '1' WHERE session_id = '".$pearDB->escape($sid)."'");
+    $pearDB->query("UPDATE session SET update_acl = '1' WHERE session_id = '" . $pearDB->escape($sid) . "'");
```

Comment:
Don't forget to create a db transaction when there is more than one update request.